hack/setup-local.sh

```diff
@@ -1,8 +1,9 @@
 kind create cluster --name kyverno-cluster
```

Comment:
This is missing the `--image` flag from ci

Reply:
This should also be fixed now.